cvat-sdk/cvat_sdk/core/proxies/tasks.py

```diff
@@ -257,23 +257,30 @@ def export_dataset(
         pbar: Optional[ProgressReporter] = None,
         status_check_period: Optional[int] = None,
         include_images: bool = True,
+        all_images: bool = True,
     ) -> None:
         """
         Download annotations for a task in the specified format (e.g. 'YOLO ZIP 1.0').
         """
 
         filename = Path(filename)
+        method: str
 
         if include_images:
-            endpoint = self.api.retrieve_dataset_endpoint
+            endpoint = self.api.create_dataset_export_endpoint
+            method = "POST"
+            query_params = {"format": format_name, "save_images": include_images, "all_images": all_images}
         else:
             endpoint = self.api.retrieve_annotations_endpoint
+            method = "GET"
+            query_params = {"format": format_name}
 
         Downloader(self._client).prepare_and_download_file_from_endpoint(
             endpoint=endpoint,
             filename=filename,
+            methodType = method,
```

Comment:
There is no such parameter in this function. Also, it should be built-in into the endpoint callback itself.